CHANGELOG.md

```diff
@@ -5,13 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Changed
+
+- Update README and make related changes. [#537](https://github.com/clowder-framework/clowder2/issues/537)
 
 ## [alpha-2] - 2022-11-28
 
 ### Added
+
 - Use Keycloak for user management, JWT tokens and federated identity.
 - Initial implementation of file versioning and nested folders within datasets.
 - Initial implementation of metadata entries and metadata definitions.
 - Initial implementation of search using Elasticsearch.
 - Improvements to overall UI and UX. Standardized components and widgets used.
 - Helm charts for Kubernetes deployment.
+
+[unreleased]: https://github.com/clowder-framework/clowder2/compare/v2.0.0-alpha.2...HEAD
+
+[alpha-2]: https://github.com/clowder-framework/clowder2/releases/tag/v2.0.0-alpha.2
```

Comment:
I like it how we include tags.. maybe we should start tagging more often? @lmarini

Reply:
yes, we should